packages/bitcore-node/src/providers/chain-state/index.ts

```diff
@@ -11,6 +11,10 @@ class ChainStateProxy implements CSP.ChainStateProvider {
     return services[chain];
   }
 
+  streamUnspentsByBlock(params: CSP.GetUnspentsFromBlockParams) {
```

Comment:
Thanks for this! :)

I'd prefer `streamCoinsByBlock` or `streamCoinsForBlock`, since this implies to me the coins won't be spent

Reply:
Done.
Please check :)